src/FileConventions/Library.fs

```diff
@@ -60,3 +60,21 @@ let DetectAsteriskInPackageReferenceItems(fileInfo: FileInfo) =
         )
 
     asteriskInPackageReference.IsMatch fileText
+
+let DetectMissingVersionsInNugetPackageReferences(fileInfo: FileInfo) =
+    assert (fileInfo.FullName.EndsWith(".fsx"))

```

Comment:
@realmarv nit: you don't need 4 parens for this line, 2 is enough

Reply:
Done